src/jimgw/run/library/IMRPhenomPv2_standard_cbc.py

```diff
@@ -66,18 +57,12 @@ def __init__(
         dL_range: tuple[float, float],
         t_c_range: tuple[float, float],
         phase_c_range: tuple[float, float],
-        psi_prior: tuple[float, float],
-        ra_prior: tuple[float, float],
+        psi_range: tuple[float, float],
+        ra_range: tuple[float, float],
+        dec_range: tuple[float, float],
+        **kwargs,
```

Comment:
The hierarchy is a bit more clear now, so inherited attribute should safely goes through kwargs.